Content.Server/Lathe/LatheSystem.cs

```diff
@@ -98,13 +99,23 @@ private void OnComponentInit(EntityUid uid, LatheComponent component, ComponentI
 
         /// <summary>
         /// When someone tries to use an item on the lathe,
-        /// insert it if it's a stack and fits inside
+        /// insert it if it's a stack and fits inside and doesn't have anything stuck to it
         /// </summary>
         private void OnInteractUsing(EntityUid uid, LatheComponent component, InteractUsingEvent args)
         {
             if (!TryComp<MaterialStorageComponent>(uid, out var storage) || !TryComp<MaterialComponent>(args.Used, out var material))
                 return;
 
+            var stickyCompCheck = new StickyComponentTestAttemptEvent(args.User);
+            RaiseLocalEvent(uid , stickyCompCheck);
+            if (stickyCompCheck.Cancelled)
+                return;
+
+            var hasEntityStuckOnCompCheck = new HasEntityStuckOnComponentTestAttemptEvent(args.User);
+            RaiseLocalEvent(uid ,hasEntityStuckOnCompCheck);
+            if (hasEntityStuckOnCompCheck.Cancelled)
+                return;
+
```

Comment:
Lathes and stacks should not be raising this. They should be calling into sticky system.